src/main/java/com/epam/test_generator/services/LoginService.java

```diff
@@ -78,13 +79,7 @@ public void checkPassword(LoginUserDTO loginUserDTO, HttpServletRequest request)
     public String getLoginJWTToken(LoginUserDTO loginUserDTO) {
 
         User user = userService.getUserByEmail(loginUserDTO.getEmail());
-        JWTCreator.Builder builder = JWT.create()
-                .withIssuer(ELEMENT_FOR_UNIQUE_TOKEN)
-                .withClaim("id", user.getId())
-                .withClaim("email", user.getEmail())
-                .withClaim("given_name", user.getName())
-                .withClaim("family_name", user.getSurname())
-                .withClaim("role", user.getRole().getName());
+        JWTCreator.Builder builder = UserBuilder.getUserBuilder(user, ELEMENT_FOR_UNIQUE_TOKEN);
```

Comment:
IMO it worth to leave code where it is

Reply:
Maybe we could extract token building related logic into a separate class `TokenBuilder` with method `String buildFor(User user)`